README.md

```diff
@@ -178,9 +177,12 @@ It should be
 
     Python 3.x.x
 
-#### (Optional) Only if using TLS or plain TCP as communication module
-We use Boost both for plain TCP and TLS communication. Please follow these
-instructions to install this library
+#### (Optional) Boost
+We use Boost both for:
+ * optional plain TCP or TLS communication
+ * optional convenience code
```

Comment:
I think you can be more specific here :)

Reply:
I was thinking of maintenance :) But you are right, I will add BlockchainView here.